topics/metagenomics/tutorials/diversity/tutorial.md

```diff
@@ -0,0 +1,410 @@
+   $$ J = H'/ln(S) $$
+                                                                                
+   H' is Shannon Weiner diversity and S is the total number of species in a sample, across all samples in dataset. 
+
+Examples for **diversity** indexes are:
+
+- **Shannons** index, which calculates the uncertainty in predicting the species identity of an individual that is selected from a community ({% cite Shannon.1948 %}).
+
+   $$ H' = -∑<sub>i=1</sub><sup>S</sup> p<sub>i</sub> \* ln(p<sub>i</sub>) $$
+    
+   pi = proportion of individuals of species i, and ln is the natural logarithm, and  S = species richness.                                                                                                                                          
+- **Berger-Parker** index, which expresses the proportional importance of the most abundant type. Highly biased by sample size and richness ({% cite Berger.1970 %} ).
+
+   $$ D = n<sub>max</sub>/N $$
+  
+   <sub>max</sub> is the abundance of the most dominant species, and N is the total number of individuals (sum of all abundances).                                                                                                                 | |
+- **Simpsons** index, which calculates the probability that two individuals selected from a community will be of the same species. Obtains small values in datasets of high diversity and large values in datasets of low diversity ({% cite SIMPSON.1949 %}).
+
+  $$ D = ∑<sub>i=1</sub><sup>S</sup> (n<sub>i</sub>/N)<sup>2</sup> $$
+  
+   ni is the number of individuals in species i, N = total number of individuals of all species, and ni/N = pi (proportion of individuals of species i), and S = species richness.
+                                                                    
+- **Inverse Simpons** index, which is the transformation of Simpsons index that increases with increasing diversity.                                                              
+- **Fishers alpha** index, which describes the relationship between the number of species and the number of individuals in those species. Parametric index of diversity that assumes that the abundance of species follows a log series distribution ({% cite Fisher.1943 %}).
+
+   $$ S\=a\*ln(1+n/a) $$
+                                                               
+   S is number of taxa, n is number of individuals and a is the Fisher's alpha.                                                                                                                                                                     
+
+**KrakenTools** is a suite of scripts designed to help Kraken users with downstream analysis of Kraken results. The Krakentool **Calculate alpha diversity** offers the possibility to calculate five different alpha diversity indexes:
```

Comment:
The output of bracken consists of all taxonomic levels: K_kingdom, P_phylum, C_class, O_order, F_family, G_genus, and S_species ; hence the alpha diversity is also calculated using all that levels. In think in most cases, alpha diversity is defined as "generally based on the number and relative abundance of taxa at some rank" ...maybe that could be explained as well.
Technically, one could add a filter step before krakentools to achieve that.

Reply:
yes a detail box sounds good, still what I said applies for Kraken and Braken